src/absil/illib.fs

```diff
@@ -452,9 +452,7 @@ type String with
     member inline x.EndsWithOrdinal(value) =
         x.EndsWith(value, StringComparison.Ordinal)
 
-module String = 
-    let indexNotFound() = raise (new KeyNotFoundException("An index for the character was not found in the string"))
```

Comment:
This deletion seems to have yielded a build error.

Reply:
It was green before you pushed the merge

Reply:
Ok I resolved that problem